views/realtime-guide.md

```diff
@@ -0,0 +1,92 @@
+# Real-time Message Guide
+
+Real-time Message on LeanCloud allows you to make a full-featured real-time chat application without having to self-host a backend service. All message history stroed in the cloud safely. Offline messages will be delivered through Push Service(APNS or FCM).
```

Comment:
Real-time Message on LeanCloud allows you...>
`LeanCloud Real-time Messaging service allows you...`

All message history stroed in the cloud safely. >
`All message history are safely stored in the cloud.`

APNS or FCM >
`[APNs](https://developer.apple.com/library/content/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/APNSOverview.html) or [FCM](https://firebase.google.com/docs/cloud-messaging/)`